docs/markdown/howtox.md

```diff
@@ -260,3 +260,26 @@ The `cmake_module_path` property is only needed for custom CMake scripts. System
 wide CMake scripts are found automatically.
 
 More information can be found [here](Dependencies.md#cmake)
+
+## Get a default not-found dependency?
+
+```meson
+null_dep = dependency('', required : false)
+```
+
+This can be used in cases where you want a default value, but might override it
```

Comment:
The example could perhaps also illustrate that 'might', showing that it's safe to use that default in targets (e.g if all the detection was under some option, and the default value ends up being used if that option was off)